app/src/main/java/com/example/mylittlelibrary/data/Game.kt

```diff
@@ -1,9 +1,11 @@
 package com.example.mylittlelibrary.data
 
+//import androidx.room.ColumnInfo
+//import androidx.room.PrimaryKey
```

Comment:
Avoid pushing commented code to Version Control.  Version control has tools that makes this unnecessary.
- You can look at history to see what used to be there.
- You can use branches for experimental features.